core/src/main/java/org/apache/accumulo/core/metadata/StoredTabletFile.java

```diff
@@ -236,7 +236,7 @@ private static String serialize(TabletFileCq tabletFileCq) {
    * returned as an empty byte array
    **/
 
-  private static byte[] encodeRow(final Key key) {
+  protected static byte[] encodeRow(final Key key) {
```

Comment:
You might want to get rid of protected and just make it package scope so it matches the same scope as TableFileCqMetadataGson